rnd/autogpt_server/autogpt_server/util/service.py

```diff
@@ -87,6 +87,11 @@ def run(self):
         # Run the main service (if it's not implemented, just sleep).
         self.run_service()
 
+    def cleanup(self):
+        if self.use_db:
+            logger.info("⏳ Disconnecting DB")
+            self.run_and_wait(db.disconnect())
```

Comment:
nit: Don't we still need to call super().cleanup() ? like the previous one (it's no-op though).

Reply:
Yeah I left it because `AppProcess.cleanup()` is empty.